lib/pavilion/commands/_run.py

```diff
@@ -234,9 +234,10 @@ def _run(self, test: TestRun):
         except TestRunError as err:
             # An unexpected TestRunError
             test.status.set(STATES.RUN_ERROR, err)
+            return
         except TimeoutError:
             # This is expected
-            pass
+            return
```

Comment:
is this adding a status to the test like the other exceptions? If not, can you add one?